tests/e2e/query_test.go

```diff
@@ -56,6 +57,20 @@ func (s *IntegrationTestSuite) queryAtomOneTx(endpoint, txHash string, msgResp c
 	return int(resp.TxResponse.Height), nil
 }
 
+func (s *IntegrationTestSuite) queryCoreDAOsParams(endpoint string) (coredaostypes.QueryParamsResponse, error) { //nolint:unused
+	body, err := httpGet(fmt.Sprintf("%s/atomone/coredaos/v1/params", endpoint))
+	if err != nil {
+		return coredaostypes.QueryParamsResponse{}, fmt.Errorf("failed to execute HTTP request: %w", err)
+	}
+
+	var params coredaostypes.QueryParamsResponse
+	if err := s.cdc.UnmarshalJSON(body, &params); err != nil {
+		return coredaostypes.QueryParamsResponse{}, err
+	}
+
+	return params, nil
+}
```

Comment:
Let's avoid having query methods of `IntegrationTestSuite` that return an error. 

If an error happens, let's just use `s.Require().NoError(err, "reason")` or just `s.Require().NoError(err)`. So callers don't need to handle error themselves.